src/components/JobsTrainingHome/index.jsx

```diff
@@ -35,9 +35,16 @@ export default function JobsTrainingHome() {
           </Col>
         </Row>
         <Link to="/opportunities">
-          <h6 className="opportunitiesButton">
-            {t("home.buttons.opportunitiesBtn")} <span>&#8594;</span>
-          </h6>
+          <Row className="opportunitiesButton">
+            <h6>
+              {t("home.buttons.opportunitiesBtn")}
```

Comment:
yeah it looks like you forgot to delete the arrow thing

Reply:
sorry I guess I am bit drunk from all the coding :/

Reply:
changed it